packages/steiger/README.md

```diff
@@ -70,43 +70,41 @@ export default defineConfig([
 
 <details>
   <summary>Comprehensive showcase of the config file syntax</summary>
-
-```javascript
-// ./steiger.config.ts
-import { defineConfig } from 'steiger'
-import fsd from '@feature-sliced/steiger-plugin'
-
-export default defineConfig([
-  ...fsd.configs.recommended,
-  {
-    // ignore all mock files for all rules
-    ignores: ['**/__mocks__/**'],
-  },
-  {
-    files: ['./src/shared/**'],
-    rules: {
-      // disable public-api rule for files in /shared folder
-      'fsd/public-api': 'off',
-    },
-  },
-  {
-    files: ['./src/widgets/**'],
-    ignores: ['**/discount-offers/**'],
-    rules: {
-      // disable no-segmentless-slices rule for all widgets except /discount-offers
-      'fsd/no-segmentless-slices': 'off',
-    },
-  },
-])
-```
-
-[You can see more examples here](../../CONFIG_EXAMPLES.md)
-
+  
+  ```javascript
+    // ./steiger.config.ts
+    import { defineConfig } from 'steiger'
+    import fsd from '@feature-sliced/steiger-plugin'
+    
+    export default defineConfig([
+      ...fsd.configs.recommended,
+      {
+        // ignore all mock files for all rules
+        ignores: ['**/__mocks__/**'],
+      },
+      {
+        files: ['./src/shared/**'],
+        rules: {
+          // disable public-api rule for files in /shared folder
+          'fsd/public-api': 'off',
+        },
+      },
+      {
+        files: ['./src/widgets/**'],
+        ignores: ['**/discount-offers/**'],
+        rules: {
+          // disable no-segmentless-slices rule for all widgets except /discount-offers
+          'fsd/no-segmentless-slices': 'off',
+        },
+      },
+    ])
+  ```
+  [You can see more examples here](CONFIG_EXAMPLES.md)
 </details>
 
 ### Migration from 0.4.0
 
-Version 0.5.0 introduced a new config file format. Follow the [instructions](../../MIGRATION_GUIDE.md) to migrate your config file.
```

Comment:
That's a good point. I think there's not a lot we can do about it for now, since this README is published to npm, and there we have no MIGRATION_GUIDE.md file at all, so this link is bound to be broken until we make a documentation site